AdyenComponents/BACS Direct Debit/BACSDirectDebitComponent.swift

```diff
@@ -50,23 +59,29 @@ public final class BACSDirectDebitComponent: PaymentComponent, PresentableCompon
     public init(paymentMethod: BACSDirectDebitPaymentMethod,
                 apiContext: APIContext,
                 style: FormComponentStyle = .init(),
-                localizationParameters: LocalizationParameters? = nil) {
+                localizationParameters: LocalizationParameters? = nil,
+                configuration: Configuration? = nil) {
         self.paymentMethod = paymentMethod
         self.apiContext = apiContext
         self.style = style
         self.localizationParameters = localizationParameters
 
-        let view = BACSDirectDebitInputFormViewController(title: paymentMethod.name,
-                                                          styleProvider: style)
+        let view = BACSInputFormViewController(title: paymentMethod.name,
```

Comment:
you should wrap this in `SecuredViewController`, to blur the form when the app goes to background, also the confirmation viewController.

Reply:
Fixed